src/components/input/input.tsx

```diff
@@ -133,7 +145,11 @@ export class Input implements LabelableComponent, FormComponent, InteractiveComp
    */
   @Prop() localeFormat = false;
 
-  /** input max */
+  /**
+   * Specifies the maximum value for type "number".
+   *
+   * @mdn [max](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#max)
+   */
   @Prop({ reflect: true }) max?: number;
 
   /** watcher to update number-to-string for max */
```

Comment:
Should the watcher line be included in the PR for consistency?

Reply:
I'm only doing public API ref. So no internal props, watchers, internal events, etc. Do you think we should update all of the comments? Seems unnecessarily time consuming